source/extensions/tracers/skywalking/skywalking_client_config.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+#include "envoy/config/trace/v3/skywalking.pb.h"
+#include "envoy/secret/secret_provider.h"
+#include "envoy/server/factory_context.h"
+#include "envoy/server/tracer_config.h"
+
+#include "absl/synchronization/mutex.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace Tracers {
+namespace SkyWalking {
+
+class SkyWalkingClientConfig {
+public:
+  SkyWalkingClientConfig(Server::Configuration::TracerFactoryContext& context,
+                         const envoy::config::trace::v3::ClientConfig& config);
+
+  uint32_t maxCacheSize() const { return max_cache_size_; }
+
+  const std::string& service() const { return service_; }
+  const std::string& serviceInstance() const { return service_instance_; }
+
+  const std::string& backendToken() const;
+
+private:
+  uint16_t max_cache_size_{0};
```

Comment:
The type of `max_cache_size` is `google.protobuf.UInt32Value` on protobuf definition. So we should follow them.

Reply:
It was a mistake caused by my negligence. Thanks.